coordinator/manifest/manifest.go

```diff
@@ -98,22 +99,14 @@ func (m Marble) Equal(other Marble) bool {
 		return false
```

Comment:
nit: `if len(m.TLS) != len(other.TLS)` was also part of the slice comparison and can be removed